hrvisa/tests/phpunit/CRM/HRVisa/ActivityTest.php

```diff
@@ -56,10 +37,12 @@ function setUp() {
     $this->createLoggedInUser();
   }
 
+  function tearDown()
+  {
+
+  }
+
   protected static function _populateDB($perClass = FALSE, &$object = NULL) {
-    if (!parent::_populateDB($perClass, $object)) {
-      return FALSE;
-    }
     _hrvisa_phpunit_populateDB();
```

Comment:
yep strange ... well its not that strange because it doesn't make sense to be able to call _hrvisa_phpunit_populateDB from here , it should be unreachable ... but its strange that the tests don't fail when I run them . but will fix that .